### DISCUSSION TITLE
Sabayon/MocaccinoOS future

### DISCUSSION THREAD

**Discussion by ghost** [mocaccinoOS/mocaccino#57] [General]
Dear Developer's,

Will there be something from Mocaccino Linux? or just a dream?

Regards,
Lacesz

**Comment by mudler** [comment]
Hi!

What are you referring to specifically? MocaccinoOS is already available and Alpha releases can be downloaded from our release page here, see https://github.com/mocaccinoOS/mocaccino/releases/tag/v0.0.1-alpha02 for example.

Cheers!
